### DISCUSSION TITLE
Unusual concentration versus time profile in lung tissue

### DISCUSSION THREAD

**Discussion by yvkashyap** [Open-Systems-Pharmacology/Forum#1667] [Questions & Problems] [answered]
Hello,
I recently developed a PBPK model using PK Sim for a drug with a Log P of 2. My goal was to predict the concentration of the drug in lung tissue after oral administration of this drug. However, when I ran the simulations to predict the lung tissue concentrations using the default PK Sim simulations mode, I noticed that the concentration versus time profile for the lung tissue concentration (black line in the attached figure) showed an unusual pattern with no decline in concentrations. In my opinion, the simulations should show that the levels in the lung must decline. Could anyone please comment on this issue and share if they had similar experiences. 

The red line is the simulated plasma concentration and the dotted line is the observed plasma concentration from a clinical study.

![Lung Tissue Concentrations](https://user-images.githubusercontent.com/42623466/205734198-c9c6f0b7-3991-4d6c-8ac3-3a99315ac546.JPG)

**Comment by StephanSchaller** [comment]
But the red line does also not decline. Did you implement any clearance processes?

**Comment by yvkashyap** [comment]
Thanks for your reply Stephan. Yes, I did. The clearance process combines metabolism by CYP2C8, CYP3A4, and renal clearance processes. These clearance processes were implemented in the ADME section of the compound file. The reported half-life of the drug is ~19 hours. Do we need to implement additional clearance processes elsewhere in the model to account for the clearance in lung tissue?

**Comment by StephanSchaller** [comment]
--> Actually this question should be asked in the forum ;-)
But it looks like your clearance is not active (plasma is not declining either)?
If you can share your project you can attach it here as zipped file.

**Comment by yvkashyap** [comment]
Hi Stephan,

Sure, I will resubmit my query in the forum. Attached is my project in the Zip file format.

[Imatinib_Kash.zip](https://github.com/Open-Systems-Pharmacology/PK-Sim/files/10158568/Imatinib_Kash.zip)

**Comment by StephanSchaller** [comment, marked as answer]
Hi Kash,

I could not find clearance with enzymes in the simulation. The Expression of enzymes was not implemented in the model:
![image](https://user-images.githubusercontent.com/25477643/205843036-5734b57f-36a9-470b-9ab1-e5af7290779e.png)

I then added a total hepatic CL: 
![image](https://user-images.githubusercontent.com/25477643/205843093-543d8ff6-0731-414e-80a4-25d032bc9345.png)


and reduced lipophilicity from 3.3 to 3 (I thought it should be 2?):
![image](https://user-images.githubusercontent.com/25477643/205842959-7ce94259-81f2-427b-aefd-d6649cc590ea.png)

Not sure this represents the truth, but it allows matching the lung data.

Please also check ou training courses (free for academia) for all sorts of applications of PK-Sim and MoBi:
https://learn-esqlabs.talentlms.com/

Maybe there is something that can help you as well.
Best, Stephan

**Comment by yvkashyap** [comment]
Hi Stephan,

Thank you so much for your time in addressing my query.  I was under the impression that if you add the metabolizing enzymes under the "Metabolism" tab of the compound block (figure below), that will by default, account for the hepatic clearance. I was worried that if I add expression profiles to the model that will add an additional clearance pathway. Anyways, I will modify the model accordingly.

Also, thanks for letting me know about the training courses. I will review them for any updates.

Sincerely,
Kash

![image](https://user-images.githubusercontent.com/42623466/205954695-93bf035e-97aa-4bbc-bb53-47dfd7e0839c.png)
